src/components/Home/index.js

```diff
@@ -1,24 +1,53 @@
-import logo from '../../logo.svg';
-import { App, AppHeader, AppLogo, AppLink } from './styled';
+import { useState } from "react";
+import trianglify from 'trianglify';
+import LevelButton from "../Icons/LevelButton";
+import RulesButton from "../Icons/RulesButton";
+import { App, AppHeader, AppLink, AppExtras } from './styled';
 
 const Home = () => {
+  const [pattern] = useState(
+    trianglify({
+      cellSize: 200,
+      height: 3840,
+      width: 2160,
+      xColors: 'YlGn',
+      colorFunction: trianglify.colorFunctions.shadows(),
+    })
+      .toSVGTree()
+      .toString()
+  );
+  const [ rulesShown, setRulesShown ] = useState(false);
+
+  const rulesClick = () => {
+    setRulesShown(true);
+  };
+
   return (
-    <App>
+    <App background={pattern}>
+      {rulesShown && <Rules setShown={setRulesShown} />}
       <AppHeader>
-        <AppLogo src={logo} alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
+        <p className="title">
+          Dino-War
         </p>
         <AppLink
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+          href="#"
```

Comment:
href="#/levels/beginner/1"
We can go ahead and add this attribute and location.